jdi-light-html-tests/src/main/java/io/github/com/sections/LoginForm.java

```diff
@@ -1,14 +1,16 @@
 package io.github.com.sections;
 
 import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
 import com.epam.jdi.light.ui.html.elements.common.Button;
 import com.epam.jdi.light.ui.html.elements.common.TextField;
 import io.github.com.entities.User;
 
-import static io.github.com.pages.Header.*;
+import static io.github.com.pages.Header.userIcon;
 
 public class LoginForm extends Form<User> {
-	TextField name, password;
+	@UI("#name") TextField name;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5478709)